src/forwarders.js

```diff
@@ -599,6 +615,30 @@ export default function Forwarders(mpInstance, kitBlocker) {
                     });
                 }
 
+                // Process sideloaded kits
+                // TODO: Sideloading kits currently requires the use of a register method
```

Comment:
You should create a sustained engineering ticket so we can follow up on this.

Reply:
will make a stand in sustained eng ticket, while ben fleshes out actual ticket since there are a lot of web specific intricacies

Reply:
https://mparticle-eng.atlassian.net/browse/SQDSDKS-5162